particles/Renderer.cpp

```diff
@@ -75,31 +75,19 @@ void Renderer::init(score::gfx::RenderList& renderer)
     auto& n = static_cast<const Node&>(this->node);
     auto& rhi = *renderer.state.rhi;
 
-    std::cout << "started init renderer\n";
-    const TexturedMeshForParticles mesh(n.meshName);
-    /*n.mesh.myData = newmesh.myData;
-    n.mesh.mesh = newmesh.mesh;
-    auto& mesh = n.mesh;*/
-
     particleOffsets = renderer.state.rhi->newBuffer(
                 QRhiBuffer::Immutable, QRhiBuffer::VertexBuffer | QRhiBuffer::StorageBuffer, maxparticles * 4 * sizeof(float));
     SCORE_ASSERT(particleOffsets->create());
     particleSpeeds = renderer.state.rhi->newBuffer(
                 QRhiBuffer::Immutable, QRhiBuffer::StorageBuffer, maxparticles * 4 * sizeof(float));
     SCORE_ASSERT(particleSpeeds->create());
     particleControls = renderer.state.rhi->newBuffer(
-                QRhiBuffer::Immutable, QRhiBuffer::UniformBuffer, sizeof(Controls));
```

Comment:
This was a bug : uniform buffers should not be immutable ; in my case this meant that changing the controls did not work.